### DISCUSSION TITLE
Reg Slickgrid Installation with Angular 12

### DISCUSSION THREAD

**Discussion by Gbhuvanesh26** [ghiscoding/Angular-Slickgrid#896] [Q&A] [answered]
Hi, While installing latest version of Slickgrid 4.1.2 with angular 12, am facing issues while building with errors such as, 
ERROR in ./node_modules/angular-slickgrid/fesm2015/angular-slickgrid.mjs 2079:48-70
Can't import the named export 'ɵɵngDeclareNgModule' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/angular-slickgrid/fesm2015/angular-slickgrid.mjs 67:259-277
Can't import the named export 'ɵɵFactoryTarget' from non EcmaScript module (only default export is available)
ERROR in ./node_modules/angular-slickgrid/fesm2015/angular-slickgrid.mjs 511:46-69
Can't import the named export 'castObservableToPromise' from non EcmaScript module (only default export is available)

I tried various stuff, and made sure ngx/translate package is installed, in tsconfig am using es2015 coz of my other dependencies, and am not able to find any proper solution for the same, pls let me know if any changes required for these issues.

**Comment by ghiscoding** [comment]
not much help I can provide apart from saying that you should simply clone the [Angular-Slickgrid-Demos](https://github.com/ghiscoding/angular-slickgrid-demos/), they are working fine and a Prod build is run for every version released. 

There's maybe 2 things to make sure of is to use latest release of both `angular-slickgrid` and any of `@slickgrid-universal` packages (if you have any of them installed). The second thing to look into is maybe related to the last error with to the fact that it requires RxJS 7+ (it won't work with RxJS 6).

**Comment by ghiscoding** [comment, marked as answer]
Side note, if you still want to support IE11 and produce ES2015 code then you should use previous version of Angular-Slickgrid 3.x as it is written in [README - Angular Compatibility](https://github.com/ghiscoding/Angular-Slickgrid#angular-compatibility) and with that downgrade you will have to also use previous version 0.x of Slickgrid-Universal packages.
